webapp/apps/taxbrain/tests.py

```diff
@@ -29,6 +29,18 @@ def test_convert(self):
         assert ans['_II_brk2'] == exp
         assert len(ans) == 1
 
+    def test_package_up_vars_CTC(self):
+        values = {"CTC_c": [2000.0]}
+        ans = package_up_vars(values)
+        exp =  [2000.0]
+        assert ans['_CTC_c'] == exp
```

Comment:
@MattHJensen here is the updated behavior: user entry of "2000" for Child Tax Credit gives back a JSON dictionary like `values` which is converted to `ans` in the above. The actual reform dictionary will then be `{2015: {'_CTC_c': [2000.0]}}`, which will then 'broadcast' as expected, which is what Martin's test shows.